### PR TITLE
Investigations

### PR DESCRIPTION
**Pull request by willrogers**
@T-Nicholls implemented a couple of the suggestions on this branch. I'll do the id/name one next.

### CODE REVIEW

**Comment by willrogers** [comment]
@T-Nicholls I'm happy to merge this one now, any comments? I don't know why Travis hasn't reported back but the build passed.

**Comment by willrogers** [comment]
@aawdls @T-Nicholls this branch is ready to merge and when it is merged I plan to release 0.3.0.

Any final comments?

File: pytac/units.py [hunk 1]
```diff
@@ -33,6 +33,7 @@ class UnitConv(object):
     **Attributes:**
 
     Attributes:
+        id (int): The unit conversion id as it appears in the csv files.
```
Comment (T-Nicholls) [inline pytac/units.py]: `name` has now replaced `id` so this line in the attributes sections of all the unit convs should become `name (str): An identifier for the unit conversion object.`?

File: pytac/units.py [hunk 2]
```diff
@@ -102,8 +115,32 @@ def eng_to_phys(self, value):
         Returns:
             float: The result value.
         """
```
Comment (T-Nicholls) [inline pytac/units.py]: Add a raises section here

File: pytac/units.py [hunk 3]
```diff
@@ -128,31 +167,63 @@ def phys_to_eng(self, value):
...
             target (str): pytac.ENG or pytac.PHYS
 
         Returns:
-            float: The result value.
+            float: The resulting value.
 
         Raises:
-            UnitsException: invalid conversion.
+            UnitsException: If the conversion is invalid; i.e. if there are no
+                             solutions, or multiple, within conversion limits.
```
Comment (T-Nicholls) [inline pytac/units.py]: This now only raises if the unit types are not understood (invalid)

File: pytac/units.py [hunk 4]
```diff
@@ -128,31 +167,63 @@ def phys_to_eng(self, value):
         Returns:
             float: The result value.
         """
```
Comment (T-Nicholls) [inline pytac/units.py]: This now raises too

File: pytac/units.py (hunk 3 above)
Comment (willrogers) [reply pytac/units.py]: Well, no, we'll still get the exception from the other methods we called.

**Comment by aawdls** [comment]
I'm getting a UnitsException

**Pull request merged**